book/website/reproducible-research/rdm/rdm-spreadsheets.md

```diff
@@ -60,12 +60,21 @@ Headers names should be chosen with care, and when it is not perfectly clear wha
 
 You do not have to think about the order of the columns for the analysis, as it has no importance for data analysis software. You can therefore completely optimise that parameter for the data collection step.
 
-## Accepted values
 
-Especially if you work with a team on data collection, it is very important to make sure everyone will enter the same information with the same term. If we take the example above, if some people use "bluish" instead of blue, the analysis of the data will be more difficult. Discrpencies might lead to errors, for instance if the same terms could mean different things depending on who is entering the data. Dates are again a very good example as 02-03 will mean Febrary third in the USA, but March third in Europe.
+
+
+### Accepted values
+
+Especially if you work with a team on data collection, it is very important to make sure everyone will enter the same information with the same term. If we take the example above, if some people use "bluish", or "b" instead of blue, the analysis of the data will be more difficult. Discrpencies might lead to errors, expecially if the same terms could mean different things depending on who is entering the data. Dates are again a very good example as 02-03 will mean February the third in the USA, but March the second in Europe.
```

Comment:
```suggestion
Especially if you work with a team on data collection, it is very important to make sure everyone will enter the same information with the same term. 
If we take the example above, if some people use "bluish", or "b" instead of blue, the analysis of the data will be more difficult. 
Discrepancies might lead to errors, especially if the same terms could mean different things depending on who is entering the data. 
Dates are again a very good example as 02-03 will mean February the third in the USA, but March the second in Europe.
```